framework/tests/all-systems/Issue263.java

```diff
@@ -1,6 +1,9 @@
 // Test case for Issue 263:
 // https://github.com/typetools/checker-framework/issues/263
 
+// Suppression required because of Issue 742
```

Comment:
742 is a merged pull request titled "Make variables static, set in static initializer block". Should this be some other number?